packages/auth-providers/dbAuth/middleware/src/__tests__/createDbAuthMiddleware.test.ts

```diff
@@ -522,6 +522,7 @@ describe('createDbAuthMiddleware()', () => {
 
       const serverAuthContext = req.serverAuthContext.get()
       expect(serverAuthContext).toBeNull()
+      expect(res.cookies.entries()).toBe('bazinga')
```

Comment:
Haven't finished this test obviously... @dthyresson if you want to feel free to finish the test up while I sleep.

If not I'll tag you again tomorrow.